website/blog/2023-07-06-run-ferretdb-on-stackgres.md

```diff
@@ -0,0 +1,499 @@
+---
+slug: run-ferretdb-on-stackgres
+title: 'How to Run FerretDB on Top of StackGres'
+authors:
+  - alex
+  - name: Álvaro Hernández
+    title: Founder and CEO @ OnGres
+    url: https://www.linkedin.com/in/alvarohernandeztortosa/
+    image_url: https://stackgres.io/img/team/alvaro.jpg
```

Comment:
Don't we want to copy such images to our repo and use "local" links? E.g. in case if this image won't be available in the future etc.